src/Microsoft.AspNet.Razor/Parser/HtmlMarkupParser.Block.cs

```diff
@@ -710,6 +764,7 @@ private bool StartTag(Stack<Tuple<HtmlSymbol, SourceLocation>> tags, IDisposable
                 return true;
             }
             Accept(_bufferedOpenAngle);
```

Comment:
- so `<!text>` will reach here and then go where?  i.e. does this eventually end up writing `<text>` out?  hoping this is at least consistent with end tag handling
- nit: blank line after closing brace (yeah, unless next line is another closing brace or part of a larger construct)